tests/support.py

```diff
@@ -3,21 +3,33 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 import logging
 import logging.handlers
+import unittest
 import weakref
 
 try:
     from unittest2 import TestCase
 except ImportError:
     # Maybe we're running in python2.7?
     from unittest import TestCase  # NOQA
-
 from webob.dec import wsgify
 from webob import exc
 from pyramid.httpexceptions import HTTPException
 from pyramid import testing
 
 from cornice.errors import Errors
 
+try:
+    import simplejson
+
+    SIMPLEJSON = True
+except ImportError:
+    SIMPLEJSON = False
```

Comment:
this could be done only once in `utils.py`, `HAS_SIMPLEJSON = ...`

Reply:
that's right, I'll fix this